app/src/main/java/com/example/healthinspector/Fragments/ProductFinderFragment.java

```diff
@@ -110,8 +110,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             OkHttpClient client = new OkHttpClient();
             Request request = null;
             try {
+                Log.i(TAG, recommendedProduct.getBrand());
```

Comment:
It's better to add to add some string that explains what's being logged and where like:
- `Log.i(TAG, "Recommended Product Brand: " + recommendedProduct.getBrand());`
- Or if it's right after a func that's defines as funcDemo(int a) - `Log.i(TAG, "funcDemo(" + a + ")");`